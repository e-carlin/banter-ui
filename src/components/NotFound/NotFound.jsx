import React from 'react';

const NotFound = () => {
  return (
    <div className="ErrorSection">
      <h1 className="ErrorHeader">404</h1>
      <h2>You weren&apos;t supposed to see this...</h2>
    </div>
  );
};

export default NotFound;
