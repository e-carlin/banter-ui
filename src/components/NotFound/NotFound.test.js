import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';

import NotFound from './NotFound';

it('NotFound renders correctly', () => {
  const tree = renderer.create(<Router><NotFound /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
