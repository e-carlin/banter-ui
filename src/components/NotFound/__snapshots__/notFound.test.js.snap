exports[`test NotFound renders correctly 1`] = `
<div
  className="ErrorSection">
  <h1
    className="ErrorHeader">
    404
  </h1>
  <h2>
    You weren\'t supposed to see this...
  </h2>
</div>
`;
